tests/unittests/OperatorTest.cpp

```diff
@@ -2450,6 +2452,83 @@ TEST_P(InterpOnly, SparseLengthsSum) {
   EXPECT_TRUE(expected.isEqual(result));
 }
 
+/// A helper function to test quantized element-wise ops on the specific
+/// backend.
+void quantizedElementwiseOpsOnSpecificBackend(BackendKind backend, Tensor *lhs,
+                                              Tensor *rhs, Tensor *out,
+                                              glow::Kinded::Kind op) {
+  ExecutionEngine EE{backend};
+  auto &mod = EE.getModule();
+  auto *X = mod.createVariable(&lhs->getType(), "x", VisibilityKind::Public,
+                               Variable::TrainKind::None);
+  auto *Y = mod.createVariable(&rhs->getType(), "y", VisibilityKind::Public,
+                               Variable::TrainKind::None);
+  glow::Function *F;
+  Node *opNode{nullptr};
+  switch (op) {
+  case glow::Kinded::Kind::ElementAddInstKind:
+    F = mod.createFunction("addFunc");
+    opNode = F->createAdd("add", X, Y);
+    break;
+  case glow::Kinded::Kind::ElementSubInstKind:
+    F = mod.createFunction("subFunc");
+    opNode = F->createSub("sub", X, Y);
+    break;
+  case glow::Kinded::Kind::ElementMulInstKind:
+    F = mod.createFunction("mulFunc");
+    opNode = F->createMul("mul", X, Y);
+    break;
+  case glow::Kinded::Kind::ElementMinInstKind:
+    F = mod.createFunction("minFunc");
+    opNode = F->createMin("min", X, Y);
+    break;
+  case glow::Kinded::Kind::ElementMaxInstKind:
+    F = mod.createFunction("maxFunc");
+    opNode = F->createMax("max", X, Y);
+    break;
+  default:
+    F = mod.createFunction("main");
+    break;
+  }
+  auto *result = F->createSave("save", opNode);
+  Function *PF = glow::profileQuantization(F);
```

Comment:
do not profile quantization in operator test.

Reply:
thx!

Reply:
updated!